### DISCUSSION TITLE
Working with arrows / lines

### DISCUSSION THREAD

**Discussion by mickwadev** [ShapeCrawler/ShapeCrawler#224] [Q&A]
Hello :)

I'm using such lines with arrow endings.
![ArrowStyles](https://user-images.githubusercontent.com/90844390/152754480-252ff17c-3a15-4dec-b02e-05eb87882f45.png)
Is there option to read those values?
I have those shapes in slide.Shapes but they are not IConnectionShape

**Comment by AdamShakhabov** [comment]
Hi :)

No, options to read these attributes do not exist yet, but we are going to [**improve lines**](https://github.com/ShapeCrawler/ShapeCrawler/discussions/203).

Could you attach presentation from screenshot.
